Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -218,6 +219,55 @@ class CustomerCenterViewModelTests: TestCase {
         }
     }
 
+    func testAppUpdateRequiredToContactSupport_true() {
+        let mockPurchases = MockCustomerCenterPurchases()
+        let latestVersion = "3.0.0"
+        let currentVersion = "2.0.0"
+        let viewModel = CustomerCenterViewModel(
+            customerCenterActionHandler: nil,
+            currentVersionFetcher: { return currentVersion },
+            purchasesProvider: mockPurchases
+        )
+        viewModel.configuration = CustomerCenterConfigTestData.customerCenterData(
+            lastPublishedAppVersion: latestVersion,
+            shouldWarnCustomerToUpdate: true
+        )
+
+        expect(viewModel.appUpdateRequiredToContactSupport).to(beTrue())
+    }
+
+    func testAppUpdateRequiredToContactSupport_false() {
+        let mockPurchases = MockCustomerCenterPurchases()
+        let latestVersion = "3.0.0"
+        let viewModel = CustomerCenterViewModel(
+            customerCenterActionHandler: nil,
+            currentVersionFetcher: { return latestVersion },
+            purchasesProvider: mockPurchases
+        )
+        viewModel.configuration = CustomerCenterConfigTestData.customerCenterData(
+            lastPublishedAppVersion: latestVersion,
+            shouldWarnCustomerToUpdate: true
+        )
+
+        expect(viewModel.appUpdateRequiredToContactSupport).to(beFalse())
+    }
+
+    func testAppUpdateRequiredToContactSupport_false_blocked_by_config() {
```

Comment:
the mix of casing here looks a little odd to my eye

Reply:
I would maybe go with something more like 
`testAppUpdateRequiredToContactSupportReturnsFalseIfBlockedByConfig `

Reply:
Ah, sometimes I mix them when the test names get super long in my apps - I'll change them to use the normal style here :)